README.md

```diff
@@ -152,7 +147,8 @@ If you want to participate in the network by running a Raiden Service Bundle you
    - We would appreciate it if you allow us access to the monitoring interfaces
      (to do that uncomment the default values of the `CIDR_ALLOW_METRICS` and `CIDR_ALLOW_PROXY` settings).
    - We also recommend that you provide your own monitoring. The setup of which is currently out of scope of this document.
-1. Make sure, that the account, configured in `KEYSTORE_FILE`, has enough funding to register as a service operator.
+   - Please, read carefully the disclaimers for the path finding and monitoring service and uncomment the variables `XX_ACCEPT_DISCLAIMER` if you agree. Note, that the services won't start if you do not. 
+1. Make sure, that the account, configured in `KEYSTORE_FILE`, has enough funding to register as a service operator (you might want to have a look at [Registering as a RSB Provider](#installation/registering) below first). Make sure that the account is stored in `${DATA_DIR}/KEYSTORE/`.
```

Comment:
Can we be more specific as to what "enough funding" is?

Reply:
@Dominik1999 Any ideas how we can specify this statement?

Reply:
the current price will always be shown here https://goerli.etherscan.io/address/0x735722704e365c37247bb3e4ec52d6c937c54539#readContract
 We need to add the correct contract version after we deployed on mainnet though and the script will tell you. Is that sufficient?

Reply:
ok let me write something

Reply:
I will add a note in the Meta issue for that @Dominik1999